l10n/en/mozorg/about-m24.ftl

```diff
@@ -38,7 +38,7 @@ m24-about-we-fund-and = We fund and invest in the people and efforts taking tech
 
 m24-about-read-more = Read more
 m24-about-read-the-latest = Read the latest { -brand-name-mozilla } news & articles
-m24-about-14-ai-projects = 14 AI projects to watch: { -brand-name-mozilla }’s first Builders Accelerator cohort kicks off
+m24-builders-program = { -brand-name-mozilla } Builders helps developers create transformative open-source AI projects through collaborations, programming, and community
```

Comment:
It looks like we have done both `{ -brand-name-mozilla-builders }` and `{ -brand-name-mozilla } Builders`. I'll ask on Slack which one we should be doing 😅